app/Console/Commands/CheckCoverage.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Exceptions\CoverageException;
+use Illuminate\Console\Command;
+
+class CheckCoverage extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'test:coverage {threshold=80}';
```

Comment:
oooooooffff 80% coverage required, hahahahahaaaaaaaaaaaaaaa :,(

Reply:
it defaults to 80 here, but if you look at the laravel.yml file above, we are passing 60 as a param